client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -43,6 +46,67 @@ const CancelPurchaseRefundInformation = ( {
 		} );
 	};
 
+	const featuresList = ( productSlug, hasDomain ) => {
+		if ( typeof productSlug !== 'string' ) {
+			return null;
+		}
+
+		const planCancellationFeatures = getPlanCancellationFeatures( productSlug, hasDomain );
+
+		return (
+			<Fragment>
+				<div className="cancel-purchase__refund-information-description-header">
+					{ i18n.translate( 'When your %(productName)s plan expires you will loose access to:', {
+						args: {
+							productName: getName( purchase ),
+						},
+					} ) }
+				</div>
+				<ul className="cancel-purchase__refund-information--list-plan-features">
+					{ planCancellationFeatures.featureList.map( ( cancellationFeature ) => {
+						return (
+							<li key={ cancellationFeature }>
+								<Gridicon
+									className="cancel-purchase__refund-information--item-cross-small"
+									size={ 24 }
+									icon="cross-small"
+								/>
+								{ getCancellationFeatureByKey( cancellationFeature ) }
+							</li>
+						);
+					} ) }
+					{ planCancellationFeatures.andMore && (
+						<li
+							className="cancel-purchase__refund-information--item-more"
+							key="cancellationAndMore"
+						>
+							<span className="cancel-purchase__refund-information--item-more-span">
+								{ i18n.translate( 'And more' ) }
+							</span>
```

Comment:
It's probably too much for this PR since they seem to already be scattered around this file but it's better to use `translate` from the `useTranslate` hook rather than calling it directly. I can't remember all the reasons for this but I think that one of them is that if the locale changes, updating the provider will not update these translations.

Reply:
I switched to `useTranslate` in all `cancel-purchase/` files